src/components/screens/Service.js

```diff
@@ -17,108 +17,96 @@ import { getQuery, QUERY_TYPES } from '../../queries';
-                        }
-                      >
-                        <View>
-                          {item.iconName ? (
-                            <Icon
-                              name={item.iconName}
-                              size={30}
-                              style={styles.serviceIcon}
-                              iconColor={colors.lightestText}
-                            />
-                          ) : (
-                            <Image
-                              source={{ uri: item.icon }}
-                              style={styles.serviceImage}
-                              PlaceholderContent={null}
-                              resizeMode="contain"
-                            />
-                          )}
-                          <BoldText
-                            small
-                            lightest
-                            center
-                            accessibilityLabel={`${item.title} (Taste)`}
-                          >
-                            {item.title}
-                          </BoldText>
-                        </View>
-                      </TouchableOpacity>
-                    </ServiceBox>
-                  );
-                })}
-              </WrapperWrap>
-            </DiagonalGradient>
-          </View>
-        );
-      }}
-    </Query>
+  return (
+    <View>
+      {!!headlineService && (
+        <TitleContainer>
+          <Title accessibilityLabel={`${headlineService} (Überschrift)`}>{headlineService}</Title>
+        </TitleContainer>
+      )}
+      {!!headlineService && device.platform === 'ios' && <TitleShadow />}
+      <DiagonalGradient style={{ padding: normalize(14) }}>
+        <WrapperWrap spaceBetween>
+          {publicJsonFileContent.map((item, index) => {
+            return (
+              <ServiceBox
+                key={index + item.title}
+                orientation={orientation}
+                dimensions={dimensions}
+              >
+                <TouchableOpacity
+                  onPress={() =>
+                    navigation.navigate({
+                      routeName: item.routeName,
+                      params: item.params
+                    })
+                  }
+                >
+                  <View>
+                    {item.iconName ? (
+                      <Icon
+                        name={item.iconName}
+                        size={30}
+                        style={styles.serviceIcon}
+                        iconColor={colors.lightestText}
+                      />
+                    ) : (
+                      <Image
+                        source={{ uri: item.icon }}
+                        style={styles.serviceImage}
+                        PlaceholderContent={null}
+                        resizeMode="contain"
+                      />
+                    )}
+                    <BoldText small lightest center accessibilityLabel={`${item.title} (Taste)`}>
```

Comment:
these props needs caution when merging in different releases!

Reply:
ok. this cause conflcts though, whenever a release has used other props there, so it should be easy to catch.